catalog/app/utils/Config.ts

```diff
@@ -47,6 +47,7 @@ export interface ConfigJson {
   qurator?: boolean
 
   build_version?: string // not sure where this comes from
+  stackVersion: string
```

Comment:
also pls update config.js.example if you're going to keep this required (or maybe we should just remove it idk)